### DISCUSSION TITLE
'/path/to/directory' is confusing.

### DISCUSSION THREAD

**Discussion by Drallas** [alexjustesen/speedtest-tracker#704] [Q&A] [answered]
In the documentation is read:

```
volumes:
    - '/etc/localtime:/etc/localtime:ro'
    - '/path/to/directory:/config'
    - '/path/to/directory/web:/etc/ssl/web'
```

Why not providing also a volume for the config:

```
.......
volumes:
    - '/etc/localtime:/etc/localtime:ro'
    - 'speedtest-config:/config'
    - 'speedtest-config/web:/etc/ssl/web'
```

It's 'best practice' to use volumes and provides a cut and paste ready docker-compose.yaml, without the user having to figure out what the hell '/path/to/directory' means.

**Comment by alexjustesen** [comment, marked as answer]
I like the suggestion and was actually looking at the docs the other day. 

Volumes are absolutely the recommend method for storage over a mounded directory and should probably be changed in the docs soon.

**Reply by Drallas** [reply]
Do I need to take of it, or will you?

**Reply by alexjustesen** [reply]
If you can open a PR in the docs repo that'd be great. I'm backed up on issues here and with client work so things aren't getting done as quickly as I'd like.

**Reply by Drallas** [reply]
No problem but, just noticed that:

```
   - 'speedtest-config:/config'
   - 'speedtest-config:/etc/ssl/web' 
``` 

Links to the inside the container; not sure if this OK or redundancy!?

<img width="632" alt="Screenshot 2023-08-31 at 15 54 59" src="https://github.com/alexjustesen/speedtest-tracker/assets/24792888/2418941a-3a1a-4de7-875b-a44a0cad4c82">
